op-batcher/batcher/channel_builder_test.go

```diff
@@ -0,0 +1,326 @@
+package batcher
+
+import (
+	"bytes"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-node/eth"
+	"github.com/ethereum-optimism/optimism/op-node/rollup/derive"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/trie"
+
+	"github.com/stretchr/testify/suite"
+)
+
+// ChannelBuilderTestSuite encapsulates testing on the ChannelBuilder.
+type ChannelBuilderTestSuite struct {
+	suite.Suite
```

Comment:
The only thing we've standardized on in testing is using `require` instead of `assert`. IMO using `suite` is overkill here, but I also have a pretty strong preference to using stock tooling.

Reply:
Agree, may as well keep it consistent. Reverted.